docs/_includes/main-nav.html

```diff
@@ -7,8 +7,10 @@
     </button>
     <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
       <form class="form-inline">
-        <input id="algolia-input" class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
-        <!-- <button class="btn btn-light my-2 my-sm-0" id="main-search" type="button">Search</button> -->
+          <!-- <img src="/assets/images/search-icon.png" alt="search icon"> -->
```

Comment:
I'm guessing we can remove the commented lines?

Reply:
I want to leave the icon in there for now as I'll be using it later on, but good catch.